src/blog-utils/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<!--
+		THIS PAGE IS FOR DEVELOPMENT PURPOSES ONLY
+	-->
+	<head>
+		<meta charset="utf-8">
+		<title>blog-utils</title>
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" />
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/inter-ui/3.15.1/inter.min.css" integrity="sha512-LmuBiKMv0sdyc1LQk0LPrsjj3KSoVgVpAXUoFGY8Ye5Zi1mff0it3I42dkh3/NGQgtkqiHcdWOcHGUmOzYLETQ==" crossorigin="anonymous" />
```

Comment:
😱 external files!

Reply:
I note the "FOR DEVELOPMENT PURPOSES ONLY" at the top but these would be blocked by the CSP?

Reply:
It's for local dev, served by a standalone node server so it won't be blocked by anything. This page isn't bundled in the lib.